app.R

```diff
@@ -16,14 +16,26 @@ ui <- div(
   tags$script(src = "https://cdn.tailwindcss.com"),
   tags$script(src = "main.js"),
   div(
-    class = "mb-2 p-1.5",
-    span(
-      class = "text-lg",
-      "🐝",
+    class = "flex items-center justify-between mr-8 mb-2 p-1.5",
```

Comment:
I'd add `ml-10` for consistency with other cards, and also I noticed that margin on the right is off due to collapsing margins in the main content. I don't suggest to fix it now, but `mr-6` looks slightly better imo.